src/index.js

```diff
@@ -128,83 +147,38 @@ export default class SauceLabs {
             return ::this._downloadJobAsset
         }
 
-        return async (...args) => {
-            const { description, method, endpoint, servers, basePath } = PROTOCOL_MAP.get(propName)
-            const params = getParameters(description.parameters)
-            const pathParams = params.filter(p => p.in === 'path')
-
-            /**
-             * validate required url params
-             */
-            let url = endpoint
-            for (const [i, urlParam] of Object.entries(pathParams)) {
-                const param = args[i]
-                const type = urlParam.type.replace('integer', 'number')
-
-                if (typeof param !== type) {
-                    throw new Error(`Expected parameter for url param '${urlParam.name}' from type '${type}', found '${typeof param}'`)
-                }
-
-                url = url.replace(`{${urlParam.name}}`, param)
-            }
-
-            /**
-             * check for body param (as last parameter as we don't expect request
-             * parameters for non idempotent requests)
-             */
-            let bodyOption = params.find(p => p.in === 'body') || description.requestBody
-                ? args[pathParams.length]
-                : null
-
-            if (bodyOption && typeof bodyOption === 'string') {
-                bodyOption = JSON.parse(bodyOption)
-            }
+        return this._callAPI.bind(this, propName)
+    }
 
-            /**
-             * validate required options
-             */
-            const bodyMap = new Map()
-            const options = args.slice(pathParams.length)[0] || {}
-            for (const optionParam of params.filter(p => p.in === 'query')) {
-                const expectedType = optionParam.type.replace('integer', 'number')
-                const optionName = camelCase(optionParam.name)
-                const option = options[optionName]
-                const isRequired = Boolean(optionParam.required) || (typeof optionParam.required === 'undefined' && typeof optionParam.default === 'undefined')
-                if ((isRequired || option) && !isValidType(option, expectedType)) {
-                    throw new Error(`Expected parameter for option '${optionName}' from type '${expectedType}', found '${typeof option}'`)
-                }
+    async _listBuilds(username, args) {
+        const { id: userId } = await this._getUserByUsername({ username })
+        const params = { userId, ...args }
+        const { builds } = await this._callAPI('getBuildsV2', 'vdc', params)
```

Comment:
I cannot find `getBuildsV2` method in files. Will it work without method definition?